src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: $calendar-width;
+  padding: 50px 10px;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $day-cell-size;
+    width: $day-cell-size;
+
+    border: 1px solid black;
+    background-color: #eee;
+    transition: 0.5s;
```

you should specify what properties should change on transition, also it's better to store transition-duration in a variable
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```